### DISCUSSION TITLE
Table Panel: Footer with multiple aggregation functions

### DISCUSSION THREAD

**Discussion by tillsteinbach** [grafana/grafana#45184] [Dashboards, Panels & Visualizations]
A rather usual use case is the need to aggregate using different functions.
Example: The table shows a list of charging sessions for an EV. You have something like this:
<img width="1305" alt="Bildschirmfoto 2022-02-09 um 22 43 09" src="https://user-images.githubusercontent.com/3825852/153295218-bbb2efe7-c64e-452d-95bb-7e78b87b6a0d.png">

For some of the columns making a sum is a good idea. You get the total energy charged or the total costs, but for others it would make sense to have an average, e.g. the average charging power or the average consumption.

The current implementation enforces to select either the one use-case or the other as it is not possible to have two aggregation functions.
A possible implementation would be to select a global function but allow to override it for each field.

**Comment by zappaterreni** [comment]
Is there a workaround for this?

**Comment by Becker884** [comment]
@tillsteinbach gibt es für dein Dashboard weitere Informationen ? Sieht sehr interessant aus, da kann ich nicht ganz mithalten. Würde mich sehr interessieren.

**Reply by tillsteinbach** [reply]
@Becker884 schau mal hier: https://github.com/tillsteinbach/VWsFriend

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
